proto/viam/app/v1/app.proto

```diff
@@ -124,6 +126,15 @@ message Location {
   google.protobuf.Timestamp created_on = 3;
 }
 
+message CreateLocationRequest {
+  string name = 1;
+  optional string org_id = 2;
```

Comment:
Is there a reason we cannot just make this required? ListLocations has it already.

Reply:
org_id -> organization_id